--- bullet-2.81-rev2613/src/BulletCollision/CollisionDispatch/btCollisionWorld.cpp	2015-01-13 16:58:05.015703412 -0500
+++ bullet-2.81-rev2613/src/BulletCollision/CollisionDispatch/btCollisionWorld.cpp	2015-01-13 16:50:55.705574578 -0500
@@ -290,12 +290,12 @@
 
 		btConvexShape* convexShape = (btConvexShape*) collisionShape;
 		btVoronoiSimplexSolver	simplexSolver;
-#define USE_SUBSIMPLEX_CONVEX_CAST 1
+//#define USE_SUBSIMPLEX_CONVEX_CAST 1
 #ifdef USE_SUBSIMPLEX_CONVEX_CAST
 		btSubsimplexConvexCast convexCaster(castShape,convexShape,&simplexSolver);
 #else
-		//btGjkConvexCast	convexCaster(castShape,convexShape,&simplexSolver);
-		//btContinuousConvexCollision convexCaster(castShape,convexShape,&simplexSolver,0);
+    btGjkConvexCast	convexCaster(castShape,convexShape,&simplexSolver);
+    //btContinuousConvexCollision convexCaster(castShape,convexShape,&simplexSolver,0);
 #endif //#USE_SUBSIMPLEX_CONVEX_CAST
 
 		if (convexCaster.calcTimeOfImpact(rayFromTrans,rayToTrans,colObjWorldTransform,colObjWorldTransform,castResult))
